newIDE/app/src/AssetStore/AssetStoreContext.js

```diff
@@ -50,7 +50,7 @@ type AssetStoreState = {|
   authors: ?Array<Author>,
   licenses: ?Array<License>,
   searchResults: ?Array<AssetShortHeader>,
-  fetchAssetsAndFilters: () => void,
+  fetchAssetsAndFilters: (showStagingAssets: boolean) => void,
```

Comment:
Would that make sense to pass this as an object? I'm always worried of one boolean, hard to read.

Reply:
Better, could we keep this option in the context?
It's like searchText, it's an option of the search:
- what do you search (`searchText`)
- with which filters (`assetFiltersState`)
- where do you search (`environment: "staging" | null`)

Reply:
yup, I put that in the NavigationState, but it makes sense to rather exist in the context